### DISCUSSION TITLE
MILESTONE 2-5

### DISCUSSION THREAD

**Discussion by Keiven-Soto** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#266] [General]
I held weekly meetings, scrum prep, and closed PRs

**Comment by AngelCIICMorales** [comment]
I held the weekly meetings, reviewed and closed pull request.

**Comment by fernandopizarro1** [comment]
Worked on testing and providing feedback for changes on some implementations for the gamestate team.

**Comment by Joaquin939** [comment]
This week I was able to finish my issues for this milestone

**Comment by DavidCuevas3** [comment]
Finished working on implementing the multiplayer leaderboard visually. #171

**Comment by nsfrontera** [comment]
Finished all my tasks, 
Implemented the all five game mode 
Implemented extra achievements 
Finished my lecture topic task

**Comment by ianramospr** [comment]
This week I finished my issues for this milestone

**Comment by Saul-Caban-UPRM** [comment]
Finished task #164 with @frances-sepulveda and joined weekly meeting with the group. Brainstormed ideas for milestone 3.

**Comment by Adriana-Isabel** [comment]
Finished the enhancement of the Home Page and a Lecture Topic Task. Gathered ideas for the final milestone.

**Comment by PinkSylvie** [comment]
I worked on finishing my lecture topic tasks regarding the project's aggregates and invariants while assisting when I could on the review process of tasks as the milestone approaches its end.

**Comment by ebrain-mendoza121** [comment]
Attended meetings, finished ui to select bot difficulty and game type. Completed lecture topic task of services example in the project

**Comment by FrancesC8** [comment]
Closed main implementation tasks of refactoring the backend leveraging SOLID principles and completed the updating the 2.3.1 Selected fragments of the implementation section in the documentation (documentation task under review, as of now).

**Comment by RicardoJRSs** [comment]
Attended meetings and finished tasks

**Comment by sandy-marrero** [comment]
- Finalizing changes for function signatures to address M1 feedback. #195

**Comment by JadiellMV** [comment]
Finished implementing the password reset feature and added repositories to the documentation.

**Comment by rukitoyordan** [comment]
Completed tasks regarding background palette, fixed some changes specific to the domain rough sketch on the documentation as well as handled LTTs.

**Comment by AnthonyMed13** [comment]
Completed task about JWT secret key bugs and finishes documentation task

**Comment by frances-sepulveda** [comment]
Work alongside @Saul-Caban-UPRM and @rukitoyordan to finish the missing tasks for this milestone.

**Reply by Gaby-Concepcion** [reply]
Attended the final meeting for the milestone and finished my remaining issues.

**Comment by Negroni1** [comment]
Completed documentation tasks, worked on the slides, and worked on finishing milestone 2 goals of the login team.

**Comment by print-Eruki** [comment]
Finalized my issues on documentation in time for delivering the Milestone documentation, working on a final task regarding unit test for UUID. Will be finished in time.

**Comment by ilikecheese117** [comment]
Finished draw dominoes and save state issues

**Comment by Jvstinoo** [comment]
Completed my lecture topic task regarding Clean Architecture. Finished all my tasks.

**Comment by G4vp** [comment]
Added 2 new buttons that allow you to choose whether you want to play against 2 bots or 3 bots.
Added logic so bots lay sequentially after each turn.
Fixed the bug where vertical tiles were not positioned correctly and fused together. I modified the code to use a grid that reads from the matrix with the table information, placing each domino in the correct position.
